src/locales/en.json

```diff
@@ -284,6 +284,13 @@
   "nft_details_page_title": "Writing NFT",
   "nft_details_page_title_collector": "Collectors",
   "nft_details_page_title_transfer": "Transfer Your NFT",
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_description": "Everyone is a content creator in this modern digital era. However different limitations still exist on internet platforms, there is no one place in which words and writings can be flown freely. Blocktrend, the most subscribed chinese blockchain media, recorded the story of the first bookstore that restore freedom and ownership to readers and authors. Welcome to the \"Web3 Bookstore\"",
+  "nft_featured_likenft1s7xk7wt5q03lsa2a87fnqfc0uny28f46s4gtnakc08qdguxlua4qxrxk7m_title": "Welcome to the first \"Web3 Bookstore\" in the world🔮",
+  "nft_featured_likenft1w8p8uwkj909ts6v78n0cn9czfrnz5g43e3w06xtgc2q4z26u2yaslvrmvs_description": "「這週大部分時間身處加州海岸的紅樹林，容許我偷懶一下，長信欠奉，附上明信片一張，閒話日常⋯⋯」It's 5 am in an early Summer day in 2022, on the long bench under the church in SF sits Mr K. When the first glimpse of sunlight shines through the arcaded windows, Mr K jotted down a few words to his distant friends. This unique postcard is now awaiting for you.",

```

Comment:
It's 5 am in an early summer day in 2022, at the long bench under the church in SF sits Mr K. When the first glimpse of sunlight shines through the arcaded windows, Mr.K jotted down a few words to his distant friends. It is a unique postcard now waiting for you to collect.

Reply:
Remove the chinese part?